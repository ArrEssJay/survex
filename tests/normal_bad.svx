; pos=fail warn=0 error=3
*fix 1 0 0 0
1 1a 2.13 012 0
1 2 - 000 0
2 3 1.00 - 0
2 * 1.00 000 0
