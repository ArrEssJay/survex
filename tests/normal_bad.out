./normal_bad.svx:4:5: error: Tape reading may not be omitted
 1 2 - 000 0
     ^
./normal_bad.svx:5:10: error: Compass reading may not be omitted except on plumbed legs
 2 3 1.00 - 0
          ^
./normal_bad.svx:6:3: error: Character "*" not allowed in station name (use *SET NAMES to set allowed characters)
 2 * 1.00 000 0
   ^

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 2 survey stations, joined by 1 leg.
There are 0 loops.
Total length of survey legs =    2.13m (   2.13m adjusted)
Total plan length of survey legs =    2.13m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from 1a at 0.00m to 1a at 0.00m)
North-South range = 2.08m (from 1a at 2.08m to 1 at 0.00m)
East-West range = 0.44m (from 1a at 0.44m to 1 at 0.00m)
   2 1-nodes.
There were 0 warning(s) and 3 error(s) - no output files produced.
